### DISCUSSION TITLE
Problem 1526B / B. I Hate 1111

### DISCUSSION THREAD

**Discussion by YagizDuman** [sonsuzus/sonsuzus#500] [Soru Cevap]
B. 1111'yi Sevmiyorum

Zaman sınırı: 1 saniye
Bellek sınırı: 256 megabayt

Bir x tam sayısı veriliyor. x'i bazı sayıları toplarak yapabilir misiniz? Bu sayılar 11, 111, 1111, 11111, vb. olabilir. (Bu sayılardan herhangi birini istediğiniz kadar kullanabilirsiniz.)

Örneğin,
33 = 11 + 11 + 11
144 = 111 + 11 + 11 + 11

Girdi
İlk satırda bir t tam sayısı verilir (1 ≤ t ≤ 10,000) — test vakalarının sayısı.
Her test vakasında tek bir satırda bir x tam sayısı verilir (1 ≤ x ≤ 10^9) — yapmanız gereken sayı.

Çıktı
Her test vakası için, eğer x'i yapabiliyorsanız "YES" (tırnak işaretleri olmadan) yazın. Aksi takdirde "NO" yazın.

"YES" ve "NO" kelimelerinin her harfini büyük veya küçük harf olarak yazabilirsiniz.

Örnek
Girdi:
3
33
144
69

Çıktı:
YES
YES
NO

ÇÖZÜM:

[ihate111 text dosyası.txt](https://github.com/user-attachments/files/17966937/ihate111.text.dosyasi.txt)

**Comment by sonsuzus** [comment]
Cevabı buraya eklemelisin.

**Comment by YagizDuman** [comment]
[ihate111.text.dosyasi.txt](https://github.com/user-attachments/files/17969358/ihate111.text.dosyasi.txt)
